lib/simulation/openfisca/mapping/index.js

```diff
@@ -7,14 +7,27 @@ var ind = require('./individu');
 var men = require('./menage');
 var fam = require('./famille');
 var foy = require('./foyerFiscal');
+var PRESTATIONS = require('../prestations');
 
 var isNotValidValue = function(value) {
     return _.isNaN(value) || _.isUndefined(value) || value === null;
 };
 
-var applyRessources = function(src, dest, schema, dateDeValeur) {
-    var agregats = {
-        anneeGlissante: {
+var getPeriods = function(dateDeValeur) {
+    return {
+        moisCourant: {
+            key: moment(dateDeValeur).format('YYYY-MM')
```

Comment:
It would be more efficient to cache the `moment` version of the date at the beginning of the function  :)
Make sure `subtract` doesn't mutate, though (or work around it by calling only `subtract(1)`.